backlogger.py

```diff
@@ -88,20 +123,25 @@ def list_issues(conf, root):
     else:
         return int(root["total_count"])
 
-
-def reminder_exists(conf, poo, msg):
+def retrieve_journals(poo):
     url = "{}/{}.json?include=journals".format(data["web"], poo["id"])
     root = json_rest("GET", url)
     if root is None:
-        sys.stderr.write("API for {} returned None, skipping reminder".format(poo["id"]))
-        return True
+        return None
     if "journals" in root["issue"]:
-        journals = root["issue"]["journals"]
-        for journal in journals:
-            if journal.get("notes", None) is None or len(journal["notes"]) == 0:
-                continue
-            if re.search(reminder_regex, journal["notes"]):
-                return True
+        return root["issue"]["journals"]
+    return None
+
+def reminder_exists(poo, journals):
+    global last_reminder, has_repeat_reminder
```

Comment:
I would avoid global variables where possible.
They are just hard to track. Especially as they are specific to a redmine ticket and not just variables that don't change once they are set.